### ISSUE TITLE
memory leak in jdbc rule when target JVM is running c3p0 pooling library

### ISSUE THREAD

**Issue by randallt** [opentracing-contrib/java-specialagent#594]
I wanted to make you aware that I opened an issue on the jdbc repo regarding a memory leak using the c3p0 JDBC pooling library:

https://github.com/opentracing-contrib/java-jdbc/issues/83

This issues causes one of our services to leak memory and OOM after a few hours of using the tracing agent. Without the agent, it is fine.

**Comment by lichenglin** [comment]
which version do you use?

I have not found such issue in 1.7.4

**Comment by randallt** [comment]
Yeah, this was on 1.7.4. It is probably linked to the usage pattern of this particular service.
